src/app/_components/project-form.tsx

```diff
@@ -157,7 +179,15 @@ export function CreateProject() {
                         id="title"
                         label="Title"
                         value={title}
-                        onChange={e => setTitle(e.target.value)}
+                        onChange={e => {
+                          setTitle(e.target.value);
+                          setUsername(
+                            e.target.value
```

Comment:
Can you stick this in `@/utils` and use this version for this task?

```ts
export const slugify = (input: string) =>
  input
    .normalize('NFD')
    .replace(/[\u0300-\u036f]/g, '')
    .toLowerCase()
    .trim()
    .replace(/\s+/g, '-')
    .replace(/[^\w-]+/g, '')
    .replace(/--+/g, '-')
    ```

Reply:
resolved in e0a2f61333e49e7a430493cbddf3426932e5e93b